src/modules/Common/components/Distinctions.module.css

```diff
@@ -0,0 +1,56 @@
+.distinctions {
+  @mixin grid 12, 11;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(280px, 1fr));
+  grid-gap: var(--gridGutterWidth);
+  justify-items: center;
+  place-content: center;
+  margin-top: var(--mXL);
+
+  &.cardList_items__isSmall {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  &.cardList_items__isBig {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (--tabletLarge) {
+  .distinctions {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+@media (--mobileExtraLarge) {
+  .distinctions {
+    grid-template-columns: repeat(1, 1fr);
+  }
+}
```

Comment:
These three distinctions add a big lot of scrolling on mobile (like the collections list below), to the point where I'm concerned with bounce rate.
Is there a way to avoid this effect? Perhaps keeping a fixed height and adding horizontal scroll? Or at least significantly decreasing the logo and text size?

Reply:
I've just re-tested on my mobile and I'm not too worried about scrolling. It doesn't seem uncomfortable to me but I hear what you're saying. I suggest to deliver now, and ask for external views and check the bounce rate evolution in the next weeks.